models/QtiRunnerInitDataBuilder.php

```diff
@@ -123,6 +123,34 @@ public function build($deliveryExecutionId, $withScores = false)
         return $init;
     }
 
+    public function getQtiTestItems(string $deliveryExecutionId): array
+    {
+        $context = $this->getServiceContext($deliveryExecutionId);
+        $testDefinition = taoQtiTest_helpers_Utils::getTestDefinition($context->getTestCompilationUri());
+        $testParts = [];
+        foreach ($testDefinition->getTestParts() as $testPart) {
+            $sections = [];
+            foreach ($testPart->getAssessmentSections() as $section) {
+                $items = [];
+                foreach ($section->getSectionParts() as $item) {
+                    $itemData = $this->qtiRunnerService->getItemData($context, $item->getHref());
```

Comment:
I'm afraid we can not use this extension code for this task, we have to somehow use qtiRunnerService directly instead, cause adding new dependency on extension-lti-test-review will be totally breaking change.

Reply:
I've moved this logic here to tao-outcome repo here https://github.com/oat-sa/extension-tao-outcome/pull/242/files#diff-21357e0b8799d4075614d38401cd14f2e765bbab11421ceac475cedb3b43e7d7 so I will close this PR